### PR TITLE
feat: #86940krgk: add hero block

### PR DESCRIPTION
**Pull request by MDI74**

### CODE REVIEW

File: components/Hero/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <span className="hero__subtitle--pink">for Junior</span>
+            {' '}
+            Frontend Developers
+          </h2>
+        </div>
+        <Image
+          className="hero__arrow"
+          src="images/hero-arrow.svg"
+          width={17}
+          height={64}
```
Comment (Yasnovskiy) [inline components/Hero/Hero.tsx:34]: I think it's better to set the sizes using styles

File: components/Hero/Hero.scss [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 28px;
+    text-align: center;
+
+    &--bold {
+      font-size: 28px;
+      line-height: 1.3;
```
Comment (Yasnovskiy) [inline components/Hero/Hero.scss:74]: here you can inherit the font size from __title. It seems that line-height does not only apply to the modifier

File: components/Layout/Layout.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren } from 'react';
+
+export default function Layout({
+  children,
+  mainClassName,
```
Comment (Yasnovskiy) [inline components/Layout/Layout.tsx:5]: Are you sure you need this? Since next you use it empty

File: styles/_globals.scss [hunk 4]
```diff
@@ -41,6 +40,7 @@ body {
 }
 
 .title-type-1 {
+  font-weight: 500;
```
Comment (Yasnovskiy) [inline styles/_globals.scss:43]: it seems that you can use the inherit property here

File: styles/_globals.scss [hunk 5]
```diff
@@ -5,7 +5,6 @@ body {
   height: 100%;
   box-sizing: border-box;
   font-family: $ff-montserrat;
-  font-weight: 500;
```
Comment (Yasnovskiy) [inline styles/_globals.scss:8]: We discussed that week that our font uses 500, I think it's worth leaving

File: components/Hero/Hero.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+
+export default function Hero() {
```
Comment (akovylyaeva) [inline components/Hero/Hero.tsx:3]: it's better to write without "default"

File: components/Layout/Layout.tsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { PropsWithChildren } from 'react';
+
+export default function Layout({
```
Comment (akovylyaeva) [inline components/Layout/Layout.tsx:3]: it's better to write without "default"

File: pages/_app.tsx [hunk 8]
```diff
@@ -1,6 +1,16 @@
 import '../styles/index.scss';
 import type { AppProps } from 'next/app';
+import { Montserrat } from 'next/font/google';
+
+const montserrat = Montserrat({
+  weight: ['400', '500', '700'],
+  subsets: ['latin', 'cyrillic'],
+});
 
 export default function App({ Component, pageProps }: AppProps) {
```
Comment (akovylyaeva) [inline pages/_app.tsx:10]: it's better to write without "default"

File: pages/index.tsx [hunk 9]
```diff
@@ -1,5 +1,10 @@
-export default function Home() {
+import Hero from '../components/Hero/Hero';
+import Layout from '../components/Layout/Layout';
+
+export default function HomePage() {
```
Comment (akovylyaeva) [inline pages/index.tsx:4]: it's better to write without "default"

**Pull request merged**